src/osemgrep/cli_ci/Ci_subcommand.ml

```diff
@@ -418,20 +418,34 @@ let partition_findings ~keep_ignored (results : OutJ.cli_match list) =
 let severity_to_int (severity : Rule.severity) =
   match severity with
   | `Experiment -> `Int 4
-  | `Warning -> `Int 1
-  | `Error -> `Int 2
+  | `Warning
+  | `Medium ->
+      `Int 1
+  | `Error
+  | `High ->
+      `Int 2
+  (* TODO: use `Int 5 here? *)
```

Comment:
@chmccreery what should be the severity integer value for Critical in the finding type?
(Note that we should really not use an int here ... it would be better to use the match_severity type which is an enum)

Reply:
I'd expect a 3 - isn't that whatwas in the interfaces PR? is there a reason you jumped to 5? I see that Experiment is 4, but don't see a 3 anywhere.

Also, I've been working on pulling severity directly out of rule_versions rather than storing it with findings anyway, so in the near future hopefully you could do whatever you want with severity (e.g. reuse match_severity type) and the app won't care. But not quite yet...

Reply:
nit: can you order these in increasing or descending order of severity for readability?